x-pack/legacy/plugins/apm/public/new-platform/plugin.tsx

```diff
@@ -51,11 +52,14 @@ const App = () => {
     <MainContainer data-test-subj="apmMainContainer" role="main">
       <UpdateBreadcrumbs routes={routes} />
       <Route component={ScrollToTopOnPathChange} />
-      <Switch>
-        {routes.map((route, i) => (
-          <ApmRoute key={i} {...route} />
-        ))}
-      </Switch>
+      {/* Check if user has permission to read from apm index. */}
```

Comment:
Nit: this could be explained in a more abstract manner. If the permissions check changes (for instance by using the Kibana privileges API) this comment will be outdated.
```
/* Check if user has the appropriate permissions to use the APM UI. */
```